ext_tables.sql

```diff
@@ -9,7 +9,7 @@ CREATE TABLE tx_securefilemount_folder
     folder      varchar(255) default ''  not null,
     folder_hash varchar(40)  DEFAULT ''  NOT NULL,
 
-    fe_group   varchar(255) default '0' not null,
+    fe_group   varchar(255) default null,
```

Comment:
Maybe rething if this column should not be named `fe_groups` in a dedicated follow up change. Not as blocker for this pull-request.

Not sure why the default / nullable change here, but buying it - assuming there is a valid reason for it.